habitat/parser.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, config):
         self.couch_server = couchdbkit.Server(config["couch_uri"])
         self.db = self.couch_server[config["couch_db"]]
 
+        # Grab the radiosonde override config.
+        self.rs_prefix = "RS_" # Default radiosonde callsign identifier.
+        self.rs_config = None
+        try:
+            self.rs_prefix = config["radiosonde_override_prefix"]
+            _rs_config = self.db[config["radiosonde_override"]]
+            self.rs_config = {'payload_configuration': _rs_config, 'id': config["radiosonde_override"]}
+            logging.debug("Loaded radiosonde override payload doc (%s)" % config["radiosonde_override"])
+        except Exception as e:
```

Comment:
There's definitely a more specific exception(s) to catch here, it's bad style to catch `Exception`.

Reply:
Fixed.